ai/priority_algorithm/priority_algorithm.py

```diff
@@ -148,9 +148,17 @@ def mutate(self, team_set: PriorityTeamSet) -> List[PriorityTeamSet]:
         """
         Mutate a single teamset into child teamsets
         """
+        algorithm = 1
```

Comment:
note;
I'll clean the way we do this at some point in the future, the direction isn't aligned with what I recommended to @Hedgemon4 to solve this problem, whichever one of you merges first will have to bully the other to some temporary consensus